### PR TITLE
docs: account kit doc on how to upgrade light account to modular account

### PR DESCRIPTION
**Pull request by denniswon**
@moldy530 @willhennessy 

Screenshot 2024-02-07 at 9 37 27 PM

# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

### Focus of this PR:
This PR focuses on upgrading Alchemy's Account Kit to support ERC-6900 in addition to ERC-4337.

### Detailed summary:
- Added a new section "Upgrading to a Modular Account" in the sidebar.
- Updated the description in the "faqs.md" file to include ERC-6900.
- Exported `chain`, `owner`, and `rpcTransport` variables in "lightAccountClient.ts".
- Updated the description in the "setup.md" file to include ERC-6900.
- Updated the description in the "nani-account.md" file to include ERC-6900.
- Updated the description in the "send-user-operations.md" file to include ERC-6900.
- Updated the description in the "custom-signer.md" file to include ERC-6900.
- Updated the description in the "dynamic.md" file to include ERC-6900.
- Updated the description in the "eoa.md" file to include ERC-6900.
- Updated the description in the "privy.md" file to include ERC-6900.
- Updated the description in the "portal.md" file to include ERC-6900.
- Updated the description in the "nft.md" file to include ERC-6900.

> The following files were skipped due to too many changes: `site/using-smart-accounts/enhanced-apis/nft.md`, `site/signers/guides/web3auth.md`, `site/signers/guides/capsule.md`, `site/signers/guides/turnkey.md`, `site/smart-accounts/index.md`, `site/using-smart-accounts/batch-user-operations.md`, `site/signers/guides/fireblocks.md`, `site/signers/guides/magic.md`, `site/signers/guides/lit.md`, `site/using-smart-accounts/simulate-user-operations.md`, `site/using-smart-accounts/enhanced-apis/token.md`, `site/smart-accounts/custom/using-your-own.md`, `site/signers/guides/arcana-auth.md`, `site/using-smart-accounts/transfer-ownership/light-account.md`, `site/smart-accounts/modular-account/upgrade-la-to-ma.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/using-smart-accounts/transfer-ownership/modular-account.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - - meta
+    - property: og:description
+      content: Follow this guide to manage ownership of a Modular Account with Account Kit, a vertically integrated stack for building apps that support ERC-4337 and ERC-6900.
+  - - meta
+    - name: twitter:title
+      content: How to Manage Ownership of a Modular Account
+  - - meta
+    - name: twitter:description
+      content: Follow this guide to manage ownership of a Modular Account with Account Kit, a vertically integrated stack for building apps that support ERC-4337 and ERC-6900.
+next:
```
Comment (moldy530) [inline site/using-smart-accounts/transfer-ownership/modular-account.md:19]: Let's remove this

File: site/using-smart-accounts/transfer-ownership/modular-account.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+The Multi Owner plugin lets your smart accounts have one or more ECDSA or SCA owners. This lets you account integrate with multiple signers at once, and supports recovering your account if one signer is lost.
+
+The Multi-Owner Plugin is able to:
+
+- Update (add, rotate, remove) owners for an MSCA.
+- Check if an address is an owner of an MSCA.
+- Show all owners of an MSCA.
+- Validate owner signatures of ERC-4337 enabled user operation transactions as well as regular transactions.
+
+Modular Account Client has exposes the following methods which allow the existing owner to add, rotate, or remove owners for an MSCA:
```
Comment (moldy530) [inline site/using-smart-accounts/transfer-ownership/modular-account.md:34]: Is this doc still a WIP?

**Review by moldy530 — COMMENTED** [review]
Upgrading content looks good to me, side bar might be a bit too long. Can we just called it, upgrading to modular account? In the doc itself we can say something along the lines of, "Using LA as an example this is how we can upgrade. This process applies to any account that supports upgrades"

File: site/using-smart-accounts/transfer-ownership/modular-account.md (hunk 2 above)
Comment (denniswon) [reply site/using-smart-accounts/transfer-ownership/modular-account.md:34]: yeah

**Comment by denniswon** [comment]
@moldy530 comments accomodated

File: site/smart-accounts/modular-account/upgrade-la-to-ma.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+```ts [example.ts]
+import { createAlchemySmartAccountClient } from "@alchemy/aa-alchemy";
+import { multiOwnerPluginActions } from "@alchemy/aa-accounts";
+
+const upgradedAccountClient = await createAlchemySmartAccountClient({
+  apiKey: "YOUR_API_KEY",
+  chain: lightAccountClient.chain,
+  account: upgradedAccount,
+}).extend(multiOwnerPluginActions);
```
Comment (moldy530) [inline site/smart-accounts/modular-account/upgrade-la-to-ma.md:60]: should probably also extend with `pluginManagerActions` and `accountLoupeActions`

**Review by moldy530 — APPROVED** [review]

**Pull request merged**